README.md

```diff
@@ -574,15 +574,21 @@ To take control over the inner organization of each field (each form row), you c
 A field template is basically a React stateless component being passed field-related props so you can structure your form row as you like:
 
 ```jsx
+
+import {Help, ErrorList} from "react-jsonschema-form/lib/components/fields/SchemaField"; // You can import also the default 'Label' from SchemaField. Below we use a custom label.
```

Comment:
I'm working on exposing the raw values along with the React components - it's a good idea and implementation is easy. So maybe we'll just leave the `Help` and `ErrorList` unimportable since the TemplateField will provide them.